cargo-paradedb/src/fixtures/io_s3_obj_store.rs

```diff
@@ -0,0 +1,302 @@
+// Copyright (c) 2023-2024 Retake, Inc.
```

Comment:
This file is nice. We don't have tooling to test benchmarking with a fake S3 bucket yet. I don't think you need to make this into a fixtures folder. Maybe just `localstack.rs` file in `src/` is sufficient.